web/template/admin/admin_tabs/maintenance.gohtml

```diff
@@ -1,18 +1,41 @@
 {{define "maintenance"}}
+    <div x-data="admin.maintenancePage()">
 
-    <h2 class="text-2xl text-1 my-auto mt-4 font-medium">Maintenance</h2>
-    <div class="form-container" x-data="admin.maintenancePage()" x-init="keepUpdated()">
-        <div class="form-container-title">Thumbnails</div>
-        <div class="form-container-body" x-init="setTimeout(() => {console.log('hi')}, 200)">
-            <button @click="generateThumbnails()" class="btn" >Regenerate All Thumbnails</button>
-            <div class="p-4" x-show="running">
-                <span class="text-1 font-semibold text-sm">Progress:</span>
+        <h2 class="text-2xl text-1 my-auto mt-4 font-medium">Maintenance</h2>
+        <div class="form-container" x-init="keepUpdated()">
+            <div class="form-container-title">Thumbnails</div>
+            <div class="form-container-body" x-init="setTimeout(() => {console.log('hi')}, 200)">
+                <button @click="generateThumbnails()" class="btn">Regenerate All Thumbnails</button>
+                <div class="p-4" x-show="running">
+                    <span class="text-1 font-semibold text-sm">Progress:</span>
                     <div class="w-full ml-2">
                         <div class="font-semibold text-sm text-4" x-text="`${Math.floor(process*100)}%`"></div>
                         <div class="w-full bg-gray-200 rounded-full h-1.5 mb-1 dark:bg-gray-700">
-                            <div class="bg-blue-600 h-1.5 rounded-full dark:bg-blue-500" :style="`width: ${process*100}%`"></div>
+                            <div class="bg-blue-600 h-1.5 rounded-full dark:bg-blue-500"
+                                 :style="`width: ${process*100}%`"></div>
```

Comment:
Sorry for nitpicking - `process` should actually be named `progress` right? 😅

Reply:
Sure, fixed that, thanks :)